README.md

```diff
@@ -1 +1,111 @@
-# CMS_Public_Use_File
+# CMS Public Use File:<br/>A Machine Learning Case Study
+
+## Table of Contents
+- [Introduction](#introduction)
+- [Dataset](#dataset)
+- [Feature Selection](#feature-selection)
+	- [Human Feature Selection](#human-featur-selection)
+	- [Computer Feature Selection](#computer-feature-selection)
+- [Model Performance](#model-performance)
+- [Conclusion](#conclusion)
+
+
+## Introduction
+The CMS data provides county and statewide Medicare and Medicaid information. There are a few target variables (Actual Per Capita Cost, Average HCC Score, etc.) to choose from and a variety of ‘per capita' explanatory variables (Ambulance Events Per 1000 Beneficiaries, Imaging Events Per 1000 Beneficiaries, etc.) Essentially, we want to see if a human can outperform a machine in predicting the Actual Per Capita Cost target variable.
+
+Here is the competition:
+- The human is only allowed to use graphics to visually pick the 5 most important explanatory variables
+- The computer is allowed to use penalized regression and decision trees to pick the 5 most important explanatory variables
+- Two multiple linear regression models will be created
+  - One with the 5 human chosen variables
+  - One with the 5 computer chosen variables
+- The species with the higher R^2 wins
+
+## Dataset
+A link to download the dataset can be found [here]()
+
+The most salient variables in the transformed dataset are:
+- Year
+- State
+- County
+- Actual Per Capita Costs – The cost of healthcare Medicare/Medicaid services per person in a given state or county
+- “….” Per 1000 Beneficiaries – Per capita estimates of healthcare service usage
+
+  ![image]()
```

Comment:
You can add images to a `README.md` as well. The only catch is that the images need to be on the actual repo. You could do this by creating a `docs/images` directory and then linking to that, or simply an `images` dir in the repo. I like `docs/images` because it gives new users some intuition on what the images pertain to (i.e. documentation).